css/css-sizing/range-percent-intrinsic-size-2.html

```diff
@@ -19,19 +19,17 @@
 
 input.b {
   height: 50%;
-  min-height: -moz-min-content;
   min-height: min-content;
   background: lime;
 }
 
 input.mb {
   max-height: 50%;
-  min-height: -moz-min-content;
   min-height: min-content;
   background: lime;
 }
 
-input.b.min-auto, input.mb.min-auto, {
+input.b.min-auto, input.mb.min-auto {
```

Comment:
There was an extra comma here (and in a couple other places), apparently didn't affect browsers though

Reply:
The extra comma makes the whole rule invalid, but looking at the test, it tests both the `min-auto` and non-`min-auto` variants, so this change is fine for me. (the alternative would be removing the min-auto tests)